### PR TITLE
Update readme

### LINKED ISSUE #37: Mention JSONResume in README
**Is this a BUG REPORT or FEATURE REQUEST?**:
bug

**What happened**:
Currently JSONResume is only mentioned on the website.

### PR DESCRIPTION
**Pull request by dbriesz**
**What this PR does / why we need it**:
Updates README in order to add mention of JSON Resume and removes blank doc files

**Which issue this PR fixes**:
fixes #37

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,5 +25,7 @@ Best place to start is our [contribution guide](/CONTRIBUTING.md).
 
 ---
 
+This project was based on [JSON Resume](https://github.com/jsonresume)
```
Comment (ss22ever) [inline README.md]: Not sure on this but do we not write "is" instead of "was"?

Comment (stp-ip) [reply README.md]: Good catch. "is" feels more natural. As a native speaker David can probably add his thoughts.

**Comment by stp-ip** [comment]
Happy to let you merge @ss22ever

**Pull request merged**